Lua/Lua.sublime-syntax

```diff
@@ -205,13 +205,13 @@ contexts:
 
     - match: local{{identifier_break}}
       scope: storage.modifier.lua
-      push: expression-list
+      push: expression-list-stmt
 
     - match: end{{identifier_break}}
       scope: invalid.illegal.unexpected-end.lua
 
     - match: (?=\S)
-      push: expression
+      push: expression-list-stmt
 
```

Comment:
Did you mean `expression-stmt` here? I don't see where it is used otherwise.

Reply:
I noticed that there was already an issue with assigning multiple global variables. That change was added along with the last test in my last commit.
```lua
    a, b = c, d
--   ^      ^ these commas didn't have a scope before
```